src/Magento/FunctionalTestingFramework/Test/Util/ActionGroupObjectExtractor.php

```diff
@@ -117,6 +118,10 @@ private function extractArguments($arguments)
             self::NODE_NAME
         );
 
+        $argData = array_filter($argData, function ($key) {
```

Comment:
Please add a comment on this filter line to explain why we are filtering out `$argData`.